pom.xml

```diff
@@ -100,13 +101,21 @@
     <version.org.freemarker>2.3.23</version.org.freemarker>
     <version.org.jboss.arquillian>1.1.11.Final</version.org.jboss.arquillian>
     <version.org.wildfly.arquillian>2.0.0.Alpha1</version.org.wildfly.arquillian>
+    <version.io.reactivex>1.0.8</version.io.reactivex>
 
     <plugin.version.org.jsonschema2pojo>0.4.14</plugin.version.org.jsonschema2pojo>
   </properties>
 
   <!-- We manage the artifacts from Commons here so that hawkular-commons-parent can be imported as a BoM by downstream Hawkular components -->
   <dependencyManagement>
     <dependencies>
+      <dependency>
```

Comment:
I put this here because  I saw that inventory API uses  jboss-logging and I imported this pom because it has the artifact version.